src/Service/RequestHandler/PostAuthToken.php

```diff
@@ -250,14 +253,16 @@ private function expireToken(Token $token) {
      *
      * @return string[]
      */
-    private function createResponse(Token $accessToken, Token $refreshToken, $expiresIn) {
+    private function createResponse(Token $accessToken, Token $refreshToken, $expiresIn)
+    {
         return array(
-            'token_type' => 'Bearer',
-            (string) $accessToken->getType() => (string) $accessToken->getTokenId(),
-            'expires_in' => $expiresIn,
+            'token_type'                      => 'Bearer',
+            (string) $accessToken->getType()  => (string) $accessToken->getTokenId(),
+            'expires_in'                      => $expiresIn,
             (string) $refreshToken->getType() => (string) $refreshToken->getTokenId(),
-            'scope' => (string) $accessToken->getScope(),
-            'user_id' => (string) $accessToken->getUserId(),
+            //todo-sg: don't create for guest user
```

Comment:
You're right, this can be done by making the refreshToken a "Token | null" param.